tiatoolbox/utils/visualization.py

```diff
@@ -773,12 +773,16 @@ def render_annotations(
         annotation geometries are decimated if appropriate.
 
         Args:
-            rgb (np.ndarray):
-                The image to render the annotation on.
-            bound_geom (Polygon):
-                A polygon representing the bounding box of the tile.
+            store (AnnotationStore):
```

Comment:
@measty Please can you update this?

Reply:
Addressed in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/624/commits/97642299ef1b93c340133f73aa706c1af950b3b1